server/handlers.go

```diff
@@ -199,6 +199,66 @@ func setBroadcastConfigHandler() http.Handler {
 	})
 }
 
+func (s *LivepeerServer) setMaxPriceForCapability() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if s.LivepeerNode.NodeType == core.BroadcasterNode {
+			maxPricePerUnit := r.FormValue("maxPricePerUnit")
+			pixelsPerUnit := r.FormValue("pixelsPerUnit")
+			currency := r.FormValue("currency")
+			pipeline := r.FormValue("pipeline")
+			modelID := r.FormValue("modelID")
+
+			if pipeline == "" || modelID == "" {
+				respond400(w, "pipeline and modelID must be set")
```

Comment:
Add return statements to prevent runtime error after failed request validation
```suggestion
				respond400(w, "pipeline and modelID must be set")
				return
```

Reply:
Agree